wot-wg-2023-draft.html

```diff
@@ -184,22 +180,24 @@ <h2>Motivation and Background</h2>
       <!-- Brief background of landscape, technology, and relationship to the Web, users, developers, implementers, and industry. -->
       <p>As a continuation of the work in <a href="https://www.w3.org/2022/07/wot-wg-2022.html">the previous charter</a>, 
         this working group is tasked with the standardization or extension of the building blocks
-        identified by the use cases resulting from the work of <a href="https://www.w3.org/groups/ig/wot">the Web of
-        Things Interest Group</a> (IG) as being important to advance the Web of Things.
-        The WoT building blocks are intended to complement and integrate to existing and emerging IoT standards using Web technologies. 
-        As a result, the building blocks focus on enabling cross-platform and cross-domain interoperability in the IoT.
-<!--
-        The proposed work items are summarized in the <a href="#scope">Scope</a> section, and
-        include <span class="todo">(KEEP in Sync) interoperability profiles for out-of-the-box interoperability;
-          protocol and payload bindings for ecosystem support; improving the descriptive power of Thing Descriptions for 
-          further use cases and improvements to the discovery mechanism;
-          </span>
+        to address the use cases and requirements identified by <a href="https://www.w3.org/groups/ig/wot">the Web of
+        Things Interest Group</a> (IG) to advance the Web of Things.
+        The WoT building blocks complement and integrate existing and emerging IoT standards using Web technologies. 
+        As a result, the building blocks enable cross-platform and cross-domain interoperability in the IoT.
+       </p><p>
+        The proposed work items are summarized in the <a href="#scope">Scope</a> section. 
+	They include interoperability profiles for out-of-the-box interoperability with other ecosystems;
+        protocol and payload bindings for the integration with other ecosystems, e.g. UPnP and Cloud systems; 
```

Comment:
where is the UPnP coming from? Use Cases document has no mention of it anywhere. It is just an example so it is maybe ok but still weird to mention this out of nowhere.

Reply:
It IS weird and wrong, thanks for noticing.
This was supposed to be OPC UA, which we discussed to be part of the next charter period.
I mistyped, and yes, I know the difference between OPC UA and UPnP. ;-)

Reply:
Btw: It WOULD be interesting to discuss whether a TD offers feature parity with a UPnP device description, see https://learn.microsoft.com/en-us/windows/win32/upnp/creating-a-device-description for example.